cocos/platform/android/CCFileUtils-android.cpp

```diff
@@ -344,6 +344,15 @@ std::vector<std::string> FileUtilsAndroid::listFiles(const std::string& dirPath)
     return fileList;
 }
 
+bool FileUtilsAndroid::removeDirectory(const std::string& path) const
+{
+    if (path.empty())
+        return false;
+    
+    bool suc = removeDirectoryJNI(path.c_str());
+    return suc;
```

Comment:
I suggest this variable can be omitted

Reply:
just `return removeDirectoryJNI(path.c_str())`

Reply:
ok, I will modify it